### PR TITLE
Added p2p stats to node stat output

### PR DESCRIPTION
**Pull request by michaeljfazio**
Adds p2p stats to node stat output (REST API) as requested in #1610

### CODE REVIEW

**Comment by AndrewWestberg** [comment]
@michaeljfazio Another common one is to get connected peers that takes a lot of processing power.
```
jcli rest v0 network stats get -h http://localhost:3100/api | grep addr | wc -l
```
Could this be added to this PR?

**Comment by michaeljfazio** [comment]
> @michaeljfazio Another common one is to get connected peers that takes a lot of processing power.
> 
> ```
> jcli rest v0 network stats get -h http://localhost:3100/api | grep addr | wc -l
> ```
> 
> Could this be added to this PR?

From what I can grok of the network peer handler, the process for getting peer info is a bit less straight forward than it is for p2p topology info. I'm not entirely confident that I can easily obtain the peer count without introducing a new performance problem. @NicolasDP do you have any additional insight on the matter?

In any case we could defer your request to another issue if it means holding up this one.

**Comment by mzabaluev** [comment]
We have updated the REST service to Actix-web 2.0. @michaeljfazio, please resolve the resulting conflicts in the `handlers.rs` file, it should be straightforward. Or comment if you need help with resolving.

**Comment by michaeljfazio** [comment]
> Acquiring the lock three times to compose a single response is excessive. Please change this to a method retrieving all data in a single go in a struct.

Fair point. Changes made 🎉

**Comment by vincenthz** [comment]
Thanks @michaeljfazio this is looking even better now !

File: jormungandr/src/rest/v0/handlers.rs [hunk 1]
```diff
@@ -156,6 +156,8 @@ async fn create_stats(context: &FullContext) -> Result<serde_json::Value, Error>
         .map_err(|e| ErrorInternalServerError(format!("Block value calculation error: {}", e)))?;
     let stats = &context.stats_counter;
     let tip_header = tip.header();
+    let stats = &context.stats_counter;
```
Comment (rinor) [inline jormungandr/src/rest/v0/handlers.rs]: Delete this please since is already present.

File: jormungandr/src/rest/v0/handlers.rs [hunk 2]
```diff
@@ -169,6 +171,9 @@ async fn create_stats(context: &FullContext) -> Result<serde_json::Value, Error>
         "lastBlockContentSize": tip_header.block_content_size(),
         "lastBlockSum": block_input_sum.0,
         "lastBlockFees": block_fee_sum.0,
+        "peerAvailableCnt": nodes_count.all_available_nodes_count().to_string(),
+        "peerUnreachableCnt": nodes_count.all_unreachable_nodes_count().to_string(),
+        "peerQuarantinedCnt": nodes_count.all_quarantined_nodes_count().to_string(),
```
Comment (rinor) [inline jormungandr/src/rest/v0/handlers.rs]: I would convert peer -> peers and lose the Cnt suffix.

Comment (michaeljfazio) [reply jormungandr/src/rest/v0/handlers.rs]: @rinor I tried to follow the convention set forth by the precedent of ```txRecvCnt``` and ```blockRecvCnt```

**Pull request merged**